src/flux/course.coffee

```diff
@@ -0,0 +1,35 @@
+_ = require 'underscore'
+
+{TaskActions, TaskStore} = require './task'
+{CrudConfig, makeSimpleStore, extendConfig} = require './helpers'
+
+CourseConfig =
+
+  loadPractice: (courseId) ->
+    @_local[courseId] ?= {}
+    @_local[courseId]['practices'] ?= []
+
+  loadedPractice: (obj, courseId) ->
+    @_local[courseId]['practices'].push(obj)
```

Comment:
This should probably also have the `@_local[courseId].practices ?= []`. Also, I think there is only 1 current practice per course (so far)